pageserver/src/page_service.rs

```diff
@@ -363,6 +365,8 @@ impl PageServerHandler {
         timeline: &dyn Timeline,
         req: &PagestreamExistsRequest,
     ) -> Result<PagestreamBeMessage> {
+        let _enter = info_span!("get_rel_exists", rel = %req.rel, req_lsn = %req.lsn).entered();
```

Comment:
I'm not a big fan of proc macros myself, so not recommending to do so, but just to show an alternative to such span creation: https://github.com/tokio-rs/tracing/blob/e0db0563064d4bb0602ac7981e4ed8fa7692b603/tracing-attributes/tests/fields.rs#L9

When an annotation like this placed, the method/function body gets wrapped with the tracing span with name = fn name, all parameters logged (unless excluded explicitly), some magical syntax to derive the vars.